app/views/profile/edit.html.erb

```diff
@@ -4,12 +4,9 @@
     <p class="intro"><%= t('.intro') %></p>
     <ul class="l-profile__intro__links list list--inline">
       <li><%= link_to t('.change_settings'), edit_user_registration_path %></li>
-      <li><%= link_to t('.feedback'), feedback.root_path(source: 'account') %></li>
```

Comment:
Could we remove the entry for this in the YML files as well?

Reply:
@davidtrussler I've removed the unnecessary yml files now, can you have another look? Thanks!